src/factories/box.ts

```diff
@@ -0,0 +1,34 @@
+import { differenceInMilliseconds, millisecondsInSecond } from 'date-fns'
+import { Graphics } from 'pixi.js'
+import { DEFAULT_TIME_COLUMN_SIZE_PIXELS } from '@/consts'
+import { RunGraphNode } from '@/models/RunGraph'
+import { waitForConfig } from '@/objects/config'
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export async function nodeBoxFactory() {
```

Comment:
A lot of these factories infer their return type. Doing this for speed now to avoid maintaining a bunch of types for functions. Planning on coming back through and resolving these linting ignores once things are stable.